apps/web-giddh/src/app/invoice/invoice.component.ts

```diff
@@ -109,6 +118,50 @@ export class InvoiceComponent implements OnInit, OnDestroy, AfterViewInit {
             this.saveLastState(tab);
         }
     }
+    /**
+     * This will get output by PO
+     *
+     * @param {boolean} event
+     * @memberof InvoiceComponent
+     */
+    public getPageHeading(): string {
```

Comment:
do the same here as well like commented in contact.component.ts

Reply:
Done